components/steps/Profile.tsx

```diff
@@ -509,6 +519,20 @@ const ProfileStep: React.FunctionComponent<Props> = props => {
             />
           </FormGroup>
 
+          <FormGroup>
+            <label>
+              Skill Level - How many hackathons have you previously attended?
+            </label>
+
+            <Select
+              name="hackathonCount"
```

Comment:
small nit: to be consistent with the rest of inputs, let's make this "hackathon-count" (again, small nit)